apps/hash-frontend/src/pages/shared/entities-table/grid-view.tsx

```diff
@@ -0,0 +1,38 @@
+import { Entity } from "@local/hash-subgraph";
+import { Grid } from "@mui/material";
+import { FunctionComponent } from "react";
+
+import { GridViewItem } from "./grid-view/grid-view-item";
+
+export const GridView: FunctionComponent<{ entities?: Entity[] }> = ({
+  entities,
+}) => {
+  return (
+    <Grid
+      container
+      sx={{
+        background: ({ palette }) => palette.common.white,
+        borderColor: ({ palette }) => palette.gray[30],
+        borderStyle: "solid",
+        borderWidth: 1,
+        borderTopWidth: 0,
+        borderBottomRightRadius: "8px",
+        borderBottomLeftRadius: "8px",
+        overflow: "hidden",
+      }}
+    >
+      {entities ? (
+        entities.map((entity, index, all) => (
+          <GridViewItem
+            key={entity.metadata.recordId.entityId}
+            entity={entity}
+            numberOfItems={all.length}
+            index={index}
+          />
+        ))
+      ) : (
+        <>skeleton</>
```

Comment:
Spooky

Reply:
Oops 😅 Added the skeleton in https://github.com/hashintel/hash/pull/3974/commits/38077238c1c1b09a25dfe4b066f922794f591891, looks like this:

<img width="1041" alt="image" src="https://github.com/hashintel/hash/assets/42802102/a5a1e3d4-3348-43ad-8f47-31479d3bb531">